packaging/debian/templates

```diff
@@ -146,11 +152,22 @@ Description: Allocated disk space
  corrupt your database when the system attempts to store pieces when no
  more physical space is actually available on your drive.
 
-Template: storagenode/bandwidth
-Type: string
-Default:
-Description: Advertised bandwidth
- How much bandwidth you can allocate to the Storj network. Be sure to
- allow for other use cases you have for your internet connection, and
- do not allocate more than the total up and download bandwidth your
- ISP can physically supply. The minimum bandwidth requirement is 2 TB.
+Template: storagenode/diskspace/wrong_format
+Type: boolean
+Default: false
+Description: Do you want to continue?
+ You did not specify an amount of space. Please specify a disk space
+ using n{G,T} notation.
+
+Template: storagenode/diskspace/below_minimum
+Type: boolean
+Default: false
+Description: Do you want to continue?
+ The required minimum amount of space is 500G.
```

Comment:
Is that the correct value? I think somewhere we require 550G (due to 10% overhead)
Best to check with Marketing/Support

Reply:
I've modified it to act as the windows installer concerning this: as I understand , 500G is the minimum "advertised" space, but the available space must allow up to 10% overhead.